resources/views/dormitory/print/print.blade.php

```diff
@@ -58,5 +58,13 @@ class="filled-in checkbox-color" />
                 </div>
             </div>
         </form>
+        <div style="margin-right: 0px" class="row">
+            <form method="POST" action="/print">
+                @csrf
+                @if(!session('message'))
```

Comment:
Why is it in a condition?

Reply:
Just for prevent spamming. In last commit its deleted.

Reply:
Hm, that's a good point.
The best way to prevent spamming would be storing the last notification time (eg. in the server side cache). That way, this could be shown for all users, for the last x hours or so. Right now, it's good as it is, we can open another issue for that.